### ISSUE TITLE
The plugin does nothing

### ISSUE THREAD

**Issue by myfirstnameispaul** [iworks/og-plugin-addon-image#1]
I installed the plugin through WordPress 6.7.1 interface and activated the plugin.

It creates a tool on the right side of the post edit page with the clickable option for Set OG:image. Clicking the button brings up the image selection. After selecting the image, nothing changes.

I save the post and the meta tags still only show the feature image.

**Comment by iworks** [comment]
Hi @myfirstnameispaul 

Did you install [OG — Better Share on Social Media](https://wordpress.org/plugins/og/) first?

his is just an addon which allow to manually set `og:image`.

Marcin

**Comment by myfirstnameispaul** [comment]
Yes, I installed that plugin before this one. That plugin is how I learned about this one.

**Comment by iworks** [comment]
@myfirstnameispaul 

I have checked, and I had no problem at all using it, and it works.

I have no idea how I can help.

M
